Tesis Doctorals en Xarxa.js

```diff
@@ -0,0 +1,277 @@
+
+{
+	"translatorID": "dd2d7c63-0c4b-4da4-b1d8-e72c7445c1e0",
+	"label": "Tesis Doctorals en Xarxa",
+	"creator": "CSUC",
+	"target": "^https?://(www\.)?(tdx\.cat|tesisenred\.net)",
```

Comment:
The regexp in JSON needs to escape the period `.` with double backslash, i.e. use
```
"target": "^https?://(www\\.)?(tdx\\.cat|tesisenred\\.net)",
```